src/middlewared/middlewared/plugins/unscheduled_reboot_alert.py

```diff
@@ -37,10 +38,22 @@ class UnscheduledRebootAlertService(Service):
         if not await middleware.call('notifier.is_freenas') and await middleware.call('notifier.failover_licensed'):
             if os.path.exists(WATCHDOG_ALERT_FILE):
                 watchdog_file = True
-                watchdog_time = os.path.getmtime(WATCHDOG_ALERT_FILE)
+                try:
+                    fd = os.open(WATCHDOG_ALERT_FILE, os.O_RDONLY)
```

Comment:
you need os.fsync to write, but to read that shouldnt not be a problem, right?

We could

```
with open(WATCHDOG_ALERT_FILE, 'rb') as f:
    watchdog_time = struct.unpack("@i", f.read(4))[0]
```

Reply:
ahh yes, that looks cleaner too.